resen/Resen.py

```diff
@@ -419,9 +421,13 @@ def add_image(self,bucket_name,docker_image):
         for x in self.valid_cores:
             if docker_image == x['version']:
                 image = 'docker.io/%s/%s:%s' % (x['org'],x['repo'],x['version'])
```

Comment:
This needs to be changed too. We need to build the "pull name" here and change this `image` parameter to simply be the version number of the resen-core.

Reply:
Agreed. image an pull_name will be built here.